tests/test_A1_csv_to_json.py

```diff
@@ -14,5 +16,29 @@ def test_create_input_json_creation_of_json_file():
     assert os.path.exists(os.path.join(elements, "test_example_create.json"))
 
 
+def test_create_json_from_csv_with_comma_separated_csv():
+    d = load_data_from_csv.create_json_from_csv(
+        CSV_TEST, "csv_comma", CSV_PARAMETERS, storage=False
+    )
+
+    assert d == {
```

Comment:
> assert d == {

You could define that dict somewhere above, so that it is easier to read. Also, that way in l. 26 and l. 38 you would have the same dictionary entry and could not make typos there. More or less like this:
```
csv_parsed = {
            "col1": {"param1": "val11", "param2": {"value": 21, "unit": "factor"}}
        }

csv_comma = {"csv_comma": csv_parsed}
csv_semicolon = {"csv_semicolon": csv_parsed}
...
assert d == csv_comma
...
assert d==csv_semicolon
```

Reply:
Done in 31018443d054cf57cea6a02bc0d090525d9ec901

Reply:
Done in 31018443d054cf57cea6a02bc0d090525d9ec901